### ISSUE TITLE
evaluate request should come after initialize request

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode#18128]
Spun off from #17902, evaluate comes before initialize

```
›From client: evaluate({"expression":"123","context":"watch"})
›From client: initialize({"adapterID":"node","pathFormat":"path","linesStartAt1":true,"columnsStartAt1":true,"supportsVariableType":true,"supportsVariablePaging":true,"supportsRunInTerminalRequest":true})
›To client: {"seq":0,"type":"response","request_seq":1,"command":"evaluate","success":false,"message":"not connected to runtime"}
›To client: {"seq":0,"type":"response","request_seq":2,"command":"initialize","success":true,"body":{"exceptionBreakpointFilters":[{"label":"All Exceptions","filter":"all","default":false},{"label":"Uncaught Exceptions","filter":"uncaught","default":true}],"supportsConfigurationDoneRequest":true,"supportsSetVariable":true,"supportsConditionalBreakpoints":true,"supportsCompletionsRequest":true,"supportsHitConditionalBreakpoints":true}}
```

> And it should call evaluate again at some point after we're attached, or the watch will say 'not connected to runtime' until a breakpoint is hit. The watch could be for some global.

**Comment by isidorn** [comment]
Fixed via 17b36a70f284c1aa0a443ccd61c3bfdf1b5b56ba

**Comment by isidorn** [comment]
@aeschli I recommend to leave it to @roblourens to verify
